python/tests/test_variable.py

```diff
@@ -497,3 +497,10 @@ def test_set_variance_convert_dtype():
 
     assert var.variances is not None
     assert var == expected
+
+
+def test_sum_mean():
+    var = sp.Variable([Dim.X], values=np.ones(5).astype(np.int64))
+    assert sp.sum(var, Dim.X) == sp.Variable(5)
+    var = sp.Variable([Dim.X], values=np.arange(6).astype(np.int64))
+    assert sp.mean(var, Dim.X) == sp.Variable(2.5)
```

Comment:
Shoud we also add a test on python types, e.g. check that if yous end an array of ints, you get a float back?

Reply:
I would be tempted to say no: In general I think it would be unmaintainable to re-test everything on the Python level. We should rather have Python tests to ensure that the correct underlying C++ code is called. "getting float back" would be a matter of testing that the class `Variable` is exposed correct. This is something we should rely on in a test of `mean` in Python.